### PR TITLE
Get ready

### PR DESCRIPTION
**Pull request by V-Ho**
#21
get ready screenshot

### CODE REVIEW

**Comment by Winkeltje84** [comment]
if you look at the assignment it looks like:
screen shot 2017-03-28 at 11 58 19

but yours looks like this:
screen shot 2017-03-28 at 11 58 37

The three need to be centralized a little bit more.
And I think the p needs to be made smaller.
But this is all styling for later

**Comment by Winkeltje84** [comment]
And don't forget "Get ready for the day" & "Know your tools" ;-)

**Comment by V-Ho** [comment]
Final styling changes:
screenshot from 2017-03-28 14-25-15

**Comment by Winkeltje84** [comment]
@miriamtocino , can you LGTM t(w)oo'?

File: app/views/getready.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (leefreemanxyz) [inline app/views/getready.html.erb]: This file should be renamed to _get_ready.html.erb (because Ruby programmers love snake_case)

**Pull request merged**